mod_reforged/hooks/items/item.nut

```diff
@@ -8,15 +8,15 @@
 
 		local ret = __original();
 		local skillsString = "";
+		local itemID = this.getInstanceID();
 		foreach (skill in ::Reforged.Items.getSkills(this))
 		{
-			local name = ::Reforged.Mod.Tooltips.parseString(format("[%s|Skill+%s]", skill.getName(), skill.ClassName));
+			local name = ::Reforged.Mod.Tooltips.parseString(format("[%s|Skill+%s,itemId:%s,itemOwner:null]", skill.getName(), skill.ClassName, itemID));
 			skillsString += format("- %s (%s, %s)\n", name, ::MSU.Text.colorPositive(skill.m.ActionPointCost), ::MSU.Text.colorNegative(skill.m.FatigueCost));
 		}
 
 		if (skillsString != "")
 		{
-			::MSU.NestedTooltips.setNestedSkillItem(this);
```

Comment:
What was this line meant to do?
I don't see `setNestedSkillItem` being used anywhere else in our code base

Reply:
This was a hacky way over at nested tooltips framework to be able to link the nested tooltips of skills to a certain item which the dummy player equipped before giving the tooltip of the skill e.g. for skills which depend on items like Shieldwall etc. But this was a half-baked solution and would break down immediately if you had nested tooltips of items within items. I've improved it over at the framework so we no longer need to do this and can instead pass an item instance Id and itemOwner directly within the nested hyperlink of skills.